views/LocationListView/LocationListView.tsx

```diff
@@ -4,16 +4,33 @@ import { Button, ListItem, Typography } from '../../.storybook/stories';
 import { LayoutProvider, NavbarWithLogout } from '../../components';
 import { LoadingWrapper } from '../../components/LoadingWrapper';
 import { useAppNavigation } from '../../hooks';
-import { resolveUserID } from '../../utils';
-import { BOT_NAMES } from '../../utils/env';
+import {
```

Comment:
[general comment] this file is VERY long, please move utils to a separate files and hooks